integration-tests/cypress/workbench/Diagram.ts

```diff
@@ -41,15 +41,21 @@ export class Diagram {
               force: true,
               view: win,
             });
+            // Move a first time to trigger nodeDrag (needed since nodeDragThreshold={1})
+            cy.getByTestId('rf__wrapper').findByTestId(sourceNodeTestId).trigger('mousemove', 500, 500, {
+              force: true,
+              view: win,
+            });
             // eslint-disable-next-line cypress/no-unnecessary-waiting
             cy.wait(400); // the time needed to calculate compatible nodes
-            cy.getByTestId('rf__wrapper').findByTestId(sourceNodeTestId).trigger('mousemove', {
-              clientX: x,
-              clientY: y,
+            //Warning : THERE IS AN ISSUE WITH THE COORDINATES HERE, the code is not doing what is meant to do ...
+            cy.getByTestId('rf__wrapper').findByTestId(sourceNodeTestId).trigger('mousemove', x, y, {
               force: true,
               view: win,
             });
-            cy.getByTestId('rf__wrapper').findByTestId(sourceNodeTestId).trigger('mouseup', { view: win });
+            cy.getByTestId('rf__wrapper').findByTestId(sourceNodeTestId).trigger('mouseup', {
+              view: win,
+            });
```

Comment:
The test is still flaky, I would rather skip that test and spend time fixing it in the next cooldown.